CHANGELOG.md

```diff
@@ -1,3 +1,183 @@
+### 3.4.0-RC1 (Oct 2018)
```

Comment:
It will be better to add `# Changelog` at the top of the file as it is not rendered properly by Doxygen

https://fossology.github.io/d4/d40/md_CHANGELOG.html

Currently the changelog file is called by the latest release number (as it is the first line of the MD file) which causes confusion.

Reply:
see this point! corrected